### PR TITLE
docs(eslint): added "Chore" tag

### LINKED ISSUE #403: Missing tag for ESLint convention
The ESLint convention includes `Chore` which is what the ESLint team actively use, however it's not present on the [README](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint).

EDIT: I'll open a PR once, all the local tests pass.

### PR DESCRIPTION
**Pull request by Berkmann18**
I added the missing tag for the ESLint commit convention.

fix #403

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -330,6 +330,12 @@
             }
           }
         },
+        "conventional-changelog-preset-loader": {
```
Comment (stevemao) [inline package-lock.json]: Not sure why this file is modified.

**Comment by stevemao** [comment]
Thanks for the PR. Could you have a look at my comment in the issue?

File: package-lock.json (hunk 1 above)
Comment (Berkmann18) [reply package-lock.json]: It was changed when I ran `npm test` which seemed odd but I decided to commit it as it was probably needed?

**Comment by Berkmann18** [comment]
@stevemao Do you want me to revert the change on the `package-lock.json`?

**Comment by hutson** [comment]
> @stevemao Do you want me to revert the change on the package-lock.json?

@Berkmann18, can you please revert the `package-lock.json` file change. It's probably benign, but I personally like to avoid mixing concerns (package lock file updates with documentation changes).

**Comment by Berkmann18** [comment]
@hutson Done.

**Pull request merged**